cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -267,17 +240,86 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 				HeaderText:  header,
 				GeneratorFunc: func(c *generator.Context) (generators []generator.Generator) {
 					return []generator.Generator{
-						NewGenConversion(arguments.OutputFileBaseName, pkg.Path, manualConversions, manualDefaults, peerPkgs, !skipDefaulters),
+						NewGenConversion(arguments.OutputFileBaseName, pkg.Path, manualConversions, peerPkgs, unsafeEquality),
 					}
 				},
 				FilterFunc: func(c *generator.Context, t *types.Type) bool {
 					return t.Name.Package == pkg.Path
 				},
 			})
 	}
+
+	// If there is a manual conversion defined between two types, exclude it
+	// from being a candidate for unsafe conversion
+	for k, v := range manualConversions {
+		if isCopyOnly(v.CommentLines) {
+			glog.V(5).Infof("Conversion function %s will not block memory copy because it is copy-only", v.Name)
+			continue
+		}
+		memoryEquivalentTypes.Skip(k.inType, k.outType)
```

Comment:
No.  The map records decisions - these are negative decisions (and override previous values).  During each type calculation, we do a DFS and stop at the first type that returns false, calculating any types from the bottom up.

So the presence of these markers will prevent any type from being used as a direct memory equivalent.